### ISSUE TITLE
Doubly counted plays

### ISSUE THREAD

**Issue by james-childress** [derek-adair/nflgame#36]
Last week (2018 week 2) several players got credited for an extra completion in their player._stats.

When looking through the game plays, it appears that the plays were entered twice into the JSON despite being the same, and when aggregated for player._stats, both were counted.

For example, looking at all completed passes to Javorius Allen gives:
```
(BAL, CIN 44, Q3, 4 and 2) (6:45) (Shotgun) J.Flacco pass short middle to J.Allen to CIN 43 for 1 yard (D.Dennard). {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 1.0, u'statId': 15, u'sequence': 2}
(BAL, BAL 38, Q3, 2 and 4) (3:21) (Shotgun) J.Flacco pass short left to J.Allen to BAL 44 for 6 yards (N.Vigil) [G.Atkins]. {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 6.0, u'statId': 15, u'sequence': 2}
(BAL, CIN 33, Q4, 1 and 10) (11:15) (Shotgun) J.Flacco pass short middle to J.Allen to CIN 17 for 16 yards (S.Williams). {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 16.0, u'statId': 15, u'sequence': 2}
(BAL, BAL 40, Q4, 3 and 10) (2:14) (Shotgun) J.Flacco pass short middle to J.Allen to BAL 47 for 7 yards (D.Dennard; H.Nickerson). {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 7.0, u'statId': 15, u'sequence': 2}
(BAL, CIN 44, Q4, 2 and 6) (1:15) (Shotgun) J.Flacco pass short middle to J.Allen to CIN 38 for 6 yards (D.Dennard). CIN-S.Williams was injured during the play. {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 6.0, u'statId': 15, u'sequence': 1}
(BAL, CIN 44, Q4, 2 and 6) (1:15) (Shotgun) J.Flacco pass short middle to J.Allen to CIN 38 for 6 yards (D.Dennard). CIN-S.Williams was injured during the play. {u'playerName': u'J.Flacco', u'clubcode': u'BAL', u'yards': 6.0, u'statId': 15, u'sequence': 4}
```
Where the last two plays are the same play entered twice, differing only in the sequence assigned.

And when looking for all completed passes thrown to Trey Burton:
```
(CHI, SEA 3, Q1, 1 and 3) (4:59) (Shotgun) M.Trubisky pass short middle to T.Burton for 3 yards, TOUCHDOWN. {u'playerName': u'M.Trubisky', u'clubcode': u'CHI', u'yards': 3.0, u'statId': 16, u'sequence': 2}
(CHI, CHI 23, Q3, 1 and 10) (10:39) (Shotgun) M.Trubisky pass short left to T.Burton to CHI 26 for 3 yards (E.Thomas). {u'playerName': u'M.Trubisky', u'clubcode': u'CHI', u'yards': 3.0, u'statId': 15, u'sequence': 1}
(CHI, CHI 46, Q3, 2 and 9) (3:46) (Shotgun) M.Trubisky pass short right to T.Burton to SEA 45 for 9 yards (J.Coleman). {u'playerName': u'M.Trubisky', u'clubcode': u'CHI', u'yards': 8.0, u'statId': 15, u'sequence': 1}
(CHI, CHI 46, Q3, 2 and 9) (3:46) (Shotgun) M.Trubisky pass short right to T.Burton to SEA 45 for 9 yards (J.Coleman). {u'playerName': u'M.Trubisky', u'clubcode': u'CHI', u'yards': 9.0, u'statId': 15, u'sequence': 4}
(CHI, CHI 10, Q4, 2 and 15) (10:04) (Shotgun) M.Trubisky pass short middle to T.Burton to CHI 15 for 5 yards (A.Calitro). {u'playerName': u'M.Trubisky', u'clubcode': u'CHI', u'yards': 5.0, u'statId': 15, u'sequence': 1}
```
the third and fourth plays are the same play counted twice, differing by one yard.

Similarly G.Bernard got credited in his player._stats for a completion even though the play was called back.

T.Lockett and J.Conner also have incorrect reception stats when compared to the count of plays they have with receptions.

**Comment by james-childress** [comment]
It seems like this isn't the only way that receptions are miscounted. If you look for completed passes to J.Conner, there are 5, put his player._stats only count 4. If you look for completed passes to T.Lockett, there are 5, put his player._stats count 6.

**Comment by mickeelm** [comment]
Have a look at #34, I think that's the same problem.

**Comment by derek-adair** [comment]
Or at least would be resolved by fixing #34
